lib/pure/bitops.nim

```diff
@@ -298,12 +362,18 @@ proc countSetBits*(x: SomeInteger): int {.inline, noSideEffect.} =
       else: result = countSetBitsNim(x.uint64)
 
 proc popcount*(x: SomeInteger): int {.inline, noSideEffect.} =
-  ## Alias for for countSetBits (Hamming weight.)
+  ## Alias for for `countSetBits <#countSetBits,SomeInteger>`_. (Hamming weight.)
   result = countSetBits(x)
 
 proc parityBits*(x: SomeInteger): int {.inline, noSideEffect.} =
   ## Calculate the bit parity in integer. If number of 1-bit
   ## is odd parity is 1, otherwise 0.
+  runnableExamples:
+    doAssert parityBits(0b0000_0000'u8) == 0
```

Comment:
likewise: just show 1st 2 lines. Less is more. Additional examples are only needed if there are edge cases to consider, which isn't the case.

ditto elsewhere.

Reply:
> Additional examples are only needed if there are edge cases to consider

[citation needed]

Reply:
rationale: procs in `std/os` tend to have lots of edge cases, which are best explained by examples (or a lengthy doc comment); it makes sense to show many examples in those cases.
here, there are no edge cases, additional examples don't add any value.